src/main/webapp/app/entities/quiz-exercise/quiz-exercise.component.html

```diff
@@ -2,13 +2,13 @@
     <h4>
         <span *ngIf="course">{{course.title}} - </span><span *ngIf="quizExercises">{{quizExercises.length}} </span>
         <span jhiTranslate="arTeMiSApp.quizExercise.home.title">Quiz Exercises</span>
-        <button *ngIf="courseId" class="btn mr-2 btn-primary float-right jh-create-entity create-quiz-exercise" [routerLink]="['new']">
+        <button *ngIf="courseId" class="btn mr-2 btn-primary float-right jh-create-entity create-quiz-exercise" [routerLink]="['quiz-exercise' , 'new']">
             <span class="fa fa-plus"></span>
```

Comment:
why did you change this?

Reply:
This change is needed also for the quiz in other case the routing does not work with the create button